src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,53 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+}
+
+export const Pagination: React.FC<Props> = ({ total, perPage, currentPage, onPageChange }) => {
+
+  const pages: number[] = getNumbers(1, Math.ceil(total / perPage));
+
+  const isLastPage = currentPage === pages.length;
+  const isFirstPage = currentPage === 1;
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage <= 1 && 'disabled'}`}>
```

Comment:
The `className` attribute uses a template string with a conditional expression. This might not work as expected because when `currentPage <= 1` is true, it will append `false` to the class list. Consider using the `classnames` library (imported as `cn`) to handle conditional class names more effectively.